app/service/data_svc.py

```diff
@@ -239,6 +240,7 @@ async def load_yaml_file(self, object_class, filename, access):
         for src in self.strip_yml(filename):
             obj = object_class.load(src)
             obj.access = access
+            obj.plugin = filename.split('/')[1]
```

Comment:
This will work because we don't need to support the server on Windows, otherwise I'd recommend using `pathlib` and `PurePath.parts` instead.